docs/api/v3.rst

```diff
@@ -1624,6 +1624,149 @@ Organization projects list
+
+    :requestheader Authorization: token to authenticate.
+
+
+Remote Repositories
+~~~~~~~~~~~~~~~~~~~
+
+Remote Repositories are the importable repositories connected via
+``GitHub``, ``GitLab`` and ``BitBucket``.
+
+
+Remote Repository listing
++++++++++++++++++++++++++
+
+
+.. http:get:: /api/v3/remote/repositories/
+
+    Retrieve a list of all Remote Repositories for the authenticated user.
+
+    **Example request**:
+
+    .. tabs::
+
+        .. code-tab:: bash
+
+            $ curl -H "Authorization: Token <token>" https://readthedocs.org/api/v3/remote/repositories/
+
+        .. code-tab:: python
+
+            import requests
+            URL = 'https://readthedocs.org/api/v3/remote/repositories/'
+            TOKEN = '<token>'
+            HEADERS = {'Authorization': f'token {TOKEN}'}
+            response = requests.get(URL, headers=HEADERS)
+            print(response.json())
+
+    **Example response**:
+
+    .. sourcecode:: json
+
+        {
+            "count": 20,
+            "next": "api/v3/remote/repositories/?limit=10&offset=10",
+            "previous": null,
+            "results": [
+                {
+                    "organization": {
+                        "avatar_url": "https://avatars.githubusercontent.com/u/12345?v=4",
+                        "created": "2019-04-29T10:00:00Z",
+                        "modified": "2019-04-29T12:00:00Z",
+                        "name": "Organization Name",
+                        "pk": 1,
+                        "slug": "organization",
+                        "url": "https://github.com/organization",
+                        "vcs_provider": "github"
+                    },
+                    "avatar_url": "https://avatars3.githubusercontent.com/u/test-rtd?v=4",
+                    "clone_url": "https://github.com/rtd/project.git",
+                    "created": "2019-04-29T10:00:00Z",
+                    "description": "This is a test project.",
+                    "full_name": "rtd/project",
+                    "html_url": "https://github.com/rtd/project",
+                    "modified": "2019-04-29T12:00:00Z",
+                    "name": "project",
+                    "pk": 1,
+                    "ssh_url": "git@github.com:rtd/project.git",
+                    "vcs": "git",
+                    "vcs_provider": "github",
+                    "admin": true
+                }
+            ]
+        }
+
+
+    The ``results`` in response is an array of remote repositories data.
+
+    :query string name: return remote repositories with matching name
+    :query string vcs: return remote repositories for specific vcs (``git``, ``svn``, ``hg`` or ``bzr``)
+    :query string vcs_provider: return remote repositories for specific vcs provider (``github``, ``gitlab`` or ``bitbucket``)
+    :query string organization: return remote repositories for specific remote organization (Using remote organization ``pk``)
```

Comment:
I think we should use the remote organization `slug`, instead. Most of the API is based/using `slug` now.